apps/src/applab/applab.js

```diff
@@ -1879,6 +1888,27 @@ Applab.onDesignModeClear = function() {
   document.getElementById('divApplab').innerHTML = Applab.levelHtml = "";
 };
 
+Applab.onDesignModeManageAssets = function(assetChosen, typeFilter) {
```

Comment:
Some jsdoc params might be useful, in particular what the expectations of assetChosen are